client/my-sites/pages/page-list.jsx

```diff
@@ -131,7 +139,9 @@ var Pages = React.createClass( {
 					} ) }
 			/>;
 		} else {
-			newPageLink = this.props.siteID ? '/page/' + this.props.siteID : '/page';
+			const { site, siteId } = this.props;
+			const sitePart = site && site.slug || siteId;
+			newPageLink = this.props.siteID ? '/page/' + sitePart : '/page';
```

Comment:
There's `siteId` and `siteID` in these lines. Which is it? :)

(Calypso tends to use `siteId`, so let's use that case)